dstrf/_dstrf.py

```diff
@@ -1,5 +1,7 @@
+from eelbrain import NDVar, combine
 import numpy as np
-from ._model import *
+from ._model import DstRF, REG_Data

```

Comment:
`import *` is discouraged in libraries because it's harder to keep track of where variables are coming from and imports more than needed